### PR TITLE
Swagger

### PR DESCRIPTION
**Pull request by mgoniszewska**

### CODE REVIEW

File: swagger_output.json [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+                  }
+                }
+              }
+            }
+          },
+          "required": false
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
```
Comment (FilipHalon) [inline swagger_output.json:520]:
```suggestion
          "201": {
            "description": "CREATED",
```

File: swagger_output.json [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+              "schema": {
+                "$ref": "#/components/schemas/Transaction"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "OK",
```
Comment (FilipHalon) [inline swagger_output.json:428]:
```suggestion
          "201": {
            "description": "CREATED",
```

File: swagger_output.json [hunk 3]
```diff
@@ -0,0 +1,707 @@
...
+          "bathroomNumber": {
+            "type": "number"
+          },
+          "floorsInBuilding": {
+            "type": "number"
+          },
+          "heating": {
+            "type": "string"
+          },
+          "otherFeaatures": {
```
Comment (FilipHalon) [inline swagger_output.json:601]:
```suggestion
          "otherFeatures": {
```

File: swagger_output.json [hunk 4]
```diff
@@ -0,0 +1,707 @@
...
+    }
+  },
+  "components": {
+    "schemas": {
+      "House": {
+        "type": "object",
+        "properties": {
+          "descriptionField": {
+            "type": "string"
+          },
```
Comment (FilipHalon) [inline swagger_output.json:564]:
```suggestion
          "owner": {
            "type": "string"
          },
          "descriptionField": {
            "type": "string"
          },
```

File: swagger_output.json [hunk 5]
```diff
@@ -0,0 +1,707 @@
...
+          "area": {
+            "type": "number"
+          },
+          "price": {
+            "type": "number"
+          },
+          "images": {
+            "type": "string"
+          },
+          "houseStatus": {
```
Comment (FilipHalon) [inline swagger_output.json:616]: I guess it could possibly be skipped for now.

File: swagger-autogen.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+const swaggerAutogen = require('swagger-autogen')();
+
+const outputFile = './swagger_output.json';
+const endpointsFiles = [
+  'src/routes/user/user.routes.js',
+  'src/routes/houses.routes.js',
+  'src/routes/transactions.routes.js',
+  './src/routes/favorites.routes.js',
```
Comment (FilipHalon) [inline swagger-autogen.js:8]:
```suggestion
  'src/routes/favorites.routes.js',
```

File: src/routes/user/validation.js [hunk 7]
```diff
@@ -2,8 +2,7 @@
 const Joi = require('@hapi/joi');
 
 const validateField = {
-  name: Joi.string().min(6).messages({
```
Comment (FilipHalon) [inline src/routes/user/validation.js:5]: It couldn't be left?

**Review by FilipHalon — APPROVED** [review]

**Review by ike-tom — APPROVED** [review]

**Pull request merged**